File: src/main/java/com/nuutrai/reactor/listeners/PlayerDamage.java
package com.nuutrai.reactor.listeners;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerItemDamageEvent;

public class PlayerDamage implements Listener {
    @EventHandler
    public void onDamage(PlayerItemDamageEvent e) {
        e.setCancelled(true);
    }
}

File: src/main/kotlin/com/nuutrai/reactor/listeners/PlayerDamage.kt
package com.nuutrai.reactor.listeners

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerItemDamageEvent

class PlayerDamage : Listener {
	@EventHandler
	fun onDamage(e: PlayerItemDamageEvent) {
		e.isCancelled = true
	}
}
